src/js/components/account/filters/programActivity/ProgramActivityFilter.jsx

```diff
@@ -65,17 +65,22 @@ export default class ProgramActivityFilter extends React.Component {
         sortedProgramActivities.forEach((programActivity) => {
             if (activities.length < this.state.shown) {
                 const label = `${programActivity.code} - ${programActivity.name}`;
-                const checked = this.props.selectedProgramActivities.includes(programActivity.id);
 
                 if (activities.length <= this.state.shown
                     && (programActivity.name !== null && programActivity.name !== '')) {
+                    // return new checkbox here
                     activities.push(
-                        <ProgramActivityItem
+                        <PrimaryCheckboxType
+                            {...programActivity}
+                            {...this.props}
+                            name={label}
+                            value={programActivity.id}
                             key={programActivity.id}
-                            programActivityID={programActivity.id}
-                            label={label}
-                            checked={checked}
-                            toggleValue={this.toggleValue} />);
+                            types={this.props.availableProgramActivities}
```

Comment:
`types` is supposed to be an object - see the JS console error message that comes up whenever the federal account page is loaded